start-mycroft.sh

```diff
@@ -131,39 +135,39 @@ function launch-background() {
     fi
 
     # Security warning/reminder for the user
-    if [[ "${1}" == "bus" ]] ; then
+    if [ "${1}" = "bus" ] ; then
         echo "CAUTION: The Mycroft bus is an open websocket with no built-in security"
         echo "         measures.  You are responsible for protecting the local port"
         echo "         8181 with a firewall as appropriate."
     fi
 
     # Launch process in background, sending logs to standard location
-    python3 -m ${_module} $_params >> /var/log/mycroft/${1}.log 2>&1 &
+    python3 -m ${_module} "$_params" >> /var/log/mycroft/"${1}".log 2>&1 &
```

Comment:
i would enclose the ENTER blob log file path in quotes (cleaner)

```
... >> "/var/log/mycroft/${1}.log" ...
```

Reply:
That's a personal preference but one I can get behind, resolved.